packages/core/src/wot-impl.ts

```diff
@@ -38,7 +38,15 @@ export default class WoTImpl {
 
     /** @inheritDoc */
     async exploreDirectory(url: string, filter?: WoT.ThingFilter): Promise<WoT.ThingDiscoveryProcess> {
-        throw new Error("not implemented");
+        const directoyThingDescription = await this.requestThingDescription(url);
+        const consumedDirectoy = await this.consume(directoyThingDescription);
+
+        const thingsProperty = await consumedDirectoy.readProperty("things");
+        const value = await thingsProperty.value();
```

Comment:
Do you think this general approach (first requesting the directory's TD, then consuming it, and reading the `things` property) makes sense here? I think the [algorithm](https://w3c.github.io/wot-scripting-api/#the-exploredirectory-method) in the Scripting API specification is not going into much detail here.

Reply:
That's the way I understand it as well 👍